packages/reakit/src/Toolbar/ToolbarFocusable.tsx

```diff
@@ -64,32 +78,40 @@ class Component extends React.Component {
 
   getToolbar = () => {
     if (typeof this.toolbar === "undefined") {
-      this.toolbar = this.getElement().closest(getSelector(Toolbar));
+      this.toolbar = this.getElement().closest(`.${Toolbar.styledComponentId}`);
```

Comment:
`getSelector` didn't work here?

Reply:
Hmm, was that updated recently? I didn’t replace it manually when started working on this component. Seems I missed something after pulling from master

Reply:
Oh, yeah XD
I just wonder if we shouldn't keep `getSelector` (and fix any possible typing issues within it) instead of using `.styledComponentId` directly, which would also let us drop the hacky code to add `styledComponentId` type.

Reply:
Thanks for note, `getSelector` works fine!